src/ppx_deriving_yojson.ml

```diff
@@ -171,18 +171,18 @@ and ser_expr_of_only_typ typ =
                  deriver (Ppx_deriving.string_of_core_type typ)
 
 (* http://desuchan.net/desu/src/1284751839295.jpg *)
-let rec desu_fold ~loc ~path f typs =
+let rec desu_fold quoter ~loc ~path f typs =
```

Comment:
I think that it would be more consistent to label this parameter (`~quoter`). Could you do this in all functions where a quoter parameter was added?

Reply:
Done! Thought about it myself, but followed the style of ppx_deriving where it's not named.